bciers/libs/components/src/auth/SessionTimeoutHandler.test.tsx

```diff
@@ -208,7 +209,34 @@ describe("SessionTimeoutHandler", () => {
     expect(mockUpdate).not.toHaveBeenCalled();
   });
 
-  it("handles session refresh failure by logging out", async () => {
+  it.only("handles session refresh failure by logging out", async () => {
+    mockUpdate.mockRejectedValue(new Error("Refresh failed"));
+
+    let capturedRefreshSession: () => Promise<void> = () => Promise.resolve();
+
+    mockCreateThrottledEventHandler.mockImplementation((refreshSession) => {
+      capturedRefreshSession = refreshSession;
+      return () => {}; // Return a no-op cleanup function
+    });
+
+    renderWithSession();
+
+    // Manually invoke the refreshSession to simulate activity
+    await capturedRefreshSession();
+
+    await waitFor(() => {
+      expect(mockSignOut).toHaveBeenCalledWith({
+        redirectTo: "http://logout.url", // NOSONAR
```

Comment:
This assertion fails because the redirectTo is '/', the fallback. When I log it, I do get the correct url